### PR TITLE
Decode a Morse code message

### PR DESCRIPTION
**Pull request by gasangw**
- Create a method to decode a Morse code character, takes a string parameter, and return the corresponding character in uppercase (e.g. decode_char(".-") returns "A").

- Create a method to decode an entire word in Morse code, takes a string parameter, and return the string representation. Every character in a word will be separated by a single space (e.g. decode_word("-- -.--") returns "MY").

- Create a method to decode the entire message in Morse code, takes a string parameter, and return the string representation. Every word will be separated by 3 spaces (e.g.

  decode("-- -.--   -. .- -- .")
 returns "MY NAME").

- Reuse methods whenever possible. Make sure that your code is DRY.

- Once your code is ready use it to decode the message from the old bottle:

      .-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...
You can use IRB for that :)

### CODE REVIEW

File: morse.rb [hunk 1]
```diff
@@ -1,15 +1,53 @@
...
+
+  morse_codes.each do |key, value|
+    return value if key == char
+  end
 end
+
+def decod_word(character)
+  word = ''
+  character.chars.each do |char|
+    word += decod_char(char)
```
Comment (Davidosky007) [inline morse.rb]: * Kindly work on this error i get when i run your code as you can see in the screenshot below your code isn't generating the right results [Check Requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/morse_code.md#:~:text=Create%20a%20method,.%2D%20%20%20%2D...%20%2D%2D%2D%20%2D..%2D%20%20%20..%2D.%20..%2D%20.%2D..%20.%2D..%20%20%20%2D%2D%2D%20..%2D.%20%20%20.%2D.%20..%2D%20%2D...%20..%20.%20...)
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @gasangw @tingamapuro04 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: morse.rb (hunk 1 above)
Comment (gasangw) [reply morse.rb]: Implemented

File: morse.rb [hunk 2]
```diff
@@ -1,15 +1,60 @@
...
+  word = ''
+  ciphered.split.each do |code|
+    word += decode_char(code)
+  end
+  word
+end
+
+def decrypt(ciphered)
+  sentence = ''
+  ciphered.split('  ').each do |word|
```
Comment (alexander16108) [inline morse.rb:58]: your team does not have a method that decodes single characters of the morse code 👍🏾  please work on this and submit again

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — COMMENTED** [review]
Hello @gasangw 👋,

_This is an additional review due to the second opinion request._

About your concern, as in the project requirements highlighted [here](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/morse_code.md#:~:text=Project%20requirements,MY%20NAME%22), you are supposed to create three methods and the Code Reviewer is right.

Code reviews expect that you follow the project requirements to the letter and therefore it is imperative to do so. If the project requires six methods, please implement the six and not less, and with the desired meaningful method names. This is necessary for us to avoid unnecessary misunderstandings in the code review process.

However, since your code works as implemented, we'll let this slide and allow you to move forward in the curriculum. But, we don't expect a repeat of this.

Your CRR will be approved ASAP, just hang in there and don't request another code on the dashboard.

Keep rocking 🚀!

**Pull request merged**